### PR TITLE
changed profile layout and added validation to phonenumber

### PR DESCRIPTION
**Pull request by shelleylv**
changed contact information in the profile (long emails created a problem with the icon) and added validation to phonenumber

### CODE REVIEW

File: src/app/components/profile/profile.html [hunk 1]
```diff
@@ -83,7 +83,7 @@
 
     <!-- PRIVATE INFO -->
 
-    <div class="col-md-8 col-md-offset-2 col-lg-6 col-lg-offset-3 col-sm-12" ng-hide="true">
+    <div class="col-md-8 col-md-offset-2 col-lg-6 col-lg-offset-3 col-sm-12" <!-- ng-hide="true" -->>
```
Comment (letterix) [inline src/app/components/profile/profile.html]: hmm, why? :) havnt pulled this request yet but wont this show the private info? (which should be disabled for now)

Comment (shelleylv) [inline src/app/components/profile/profile.html]: There is no information available to show, must have forgot to undo the changes made

File: src/app/components/profile/profile.html [hunk 2]
```diff
@@ -83,7 +83,7 @@
 
     <!-- PRIVATE INFO -->
 
-    <div class="col-md-8 col-md-offset-2 col-lg-6 col-lg-offset-3 col-sm-12" ng-hide="true">
+    <div class="col-md-8 col-md-offset-2 col-lg-6 col-lg-offset-3 col-sm-12" <!-- ng-hide="true" -->
```
Comment (letterix) [inline src/app/components/profile/profile.html]: Still looks odd here. Shouldn't have the ng-hide out-commented.

**Comment by letterix** [comment]
opening a merged version.
